libraries/Microsoft.Bot.Builder.Ai.QnA/Microsoft.Bot.Builder.Ai.QnA.csproj

```diff
@@ -43,6 +43,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Bot.Builder" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
+    <PackageReference Include="Microsoft.Recognizers.Text" Version="1.0.3" />
```

Comment:
Does this project need these packages? I think they were incorrectly added here.

Reply:
I think these were references before (at the time we created the PR) and were removed. I'll double check and remove them if not required.